src/core/qgsproject.h

```diff
@@ -500,9 +500,32 @@ class CORE_EXPORT QgsProject : public QObject, public QgsExpressionContextGenera
      */
     QStringList readListEntry( const QString &scope, const QString &key, const QStringList &def = QStringList(), bool *ok = nullptr ) const;
 
+    /**
+     * \returns entry value as string from \a scope given its \a key
```

Comment:
Returns needs to come after the param annotations so that sipify can do its full magic

Reply:
Thanks for pointing that out. I did the modifications